src/Pages/SingleProduct/SingleProduct.jsx

```diff
@@ -9,14 +9,15 @@ import { AiFillLike } from "react-icons/ai";
 import { useAuth } from "../../context/authContext";
 import { addToLike } from "../../Utils/addToLike";
 import { useLiked } from "../../context/likeContext";
+import { useWatchLater } from "../../context/watchContext";
+import { addToWatchLater } from "../../Utils/addToWatchLater";
```

Comment:
use index.js files in all the folder, and export all files there and then wherever you want to import it you can import it from there. 
Like this: 
```
import { useAuth, useLiked, useWatchLater } from "../../context";

import { addToLike, addToWatchLater } from "../../Utils"
```

this looks more clean